source/educators/how-tos/course_development/exercise_tools/create_zooming_image.rst

```diff
@@ -63,21 +63,21 @@ Create a Zooming Image Tool
      image that you want to appear magnified when the user hovers over the
      regular image.
 
-     ``https://studio.edx.org/c4x/edX/DemoX/asset/pathways_detail_01.png``
+     ``https://studio.openedx.org/c4x/openedX/DemoX/asset/pathways_detail_01.png``
 
      For example, your file name and path might be ``/static/Image1.jpg``.
 
    - Replace the following file name and path with the name and path of the
      image that you want to appear when the page opens.
 
-    ``https://studio.edx.org/c4x/edX/DemoX/asset/pathways_overview_01.png``
+    ``https://studio.openedx.org/c4x/openedX/DemoX/asset//pathways_overview_01.png``
 
      For example, your file name and path might be ``/static/Image2.jpg``.
 
    - Replace the following name and file path with the name and path of the
      JavaScript file that you downloaded from ``files.edx.org``.
```

Comment:
This link: http://files.edx.org/jqueryloupeAndLightbox.js does not work. Please file a bug with this page.